spec/controllers/users/invitations_controller_spec.rb

```diff
@@ -219,22 +219,16 @@
       end
 
       context "inviting a greeter" do
-        let(:coalition) { create(:coalition) }
-        let(:coalition_2) { create(:coalition) }
-        let(:organization) { create(:organization) }
+        let!(:org_1) { create(:organization, allows_greeters: true) }
+        let!(:org_2) { create(:organization, allows_greeters: true) }
```

Comment:
What's the significance of creating these organizations? I think this test would work fine without any orgs, so consider removing them.